src/main/java/org/mskcc/oncokb/curation/OncokbCurationApp.java

```diff
@@ -75,6 +81,23 @@ public void importOncoKbSequence() throws ApiException, IOException {
         }
     }
 
+    @PostConstruct
+    public void initFirebase() throws IOException {
+        FirebaseOptions options = new FirebaseOptions.Builder()
+            .setCredentials(
+                GoogleCredentials.fromStream(
+                    getClass()
+                        .getClassLoader()
+                        .getResource(applicationProperties.getFrontend().getFirebase().getServiceAccountCredentialsPath())
+                        .openStream()
+                )
+            )
+            .setDatabaseUrl(applicationProperties.getFrontend().getFirebase().getDatabaseUrl())
+            .build();
+
+        FirebaseApp.initializeApp(options);
+    }
```

Comment:
We use firebase in the backend, we should leave FirebaseProperties under applicationProperties and backfill the frontendProperties from applicationProperties